packages/commonwealth/client/scripts/state/api/groups/refreshMembership.ts

```diff
@@ -39,7 +39,13 @@ const refreshMembership = async ({
     groupId: r.groupId,
     topicIds: r.topicIds,
     isAllowed: r.allowed,
-    rejectReason: JSON.parse(`${r.rejectReason || {}}`),
+    rejectReason: (() => {
+      try {
+        return JSON.parse(`${r.rejectReason || {}}`);
+      } catch {
+        return 'Failed to parse JSON';
+      }
+    })(),
```

Comment:
I was getting errors when parsing `rejectReason` json from /refresh-membership, so I did this workaround. Not sure if there is some API change needed

cc: @rbennettcw @jnaviask

Reply:
On the gating PR, this was updated to:
```ts
rejectReason: r.rejectReason ? JSON.parse(r.rejectReason) : null,
```

Reply:
Gating PR was merged– fix the conflict and you'll see the latest code fix